packages/common/src/intl/locales/en/replenishment.json

```diff
@@ -2,11 +2,13 @@
   "button.delete-lines": "Delete lines",
   "button.new-shipment": "New Shipment",
   "button.save-and-confirm-status": "Confirm {{status}}",
+  "button.requested-to-suggested": "Requested to Suggested",
```

Comment:
oh no! this could go either way!
`Suggested to Requested` could mean the same thing.. as in, take the suggested and put it in requested.

maybe something like `Use Suggested`? ooh, what about `Request Suggested`?

Reply:
`Request Suggested` it is (because it was the last one 😁 )